tests/system/providers/amazon/aws/redshift/example_redshift_cluster.py

```diff
@@ -28,7 +28,11 @@
     RedshiftResumeClusterOperator,
 )
 from airflow.providers.amazon.aws.sensors.redshift_cluster import RedshiftClusterSensor
+from airflow.utils.trigger_rule import TriggerRule
+from tests.system.providers.amazon.aws.utils import set_env_id
 
+ENV_ID = set_env_id()
+DAG_ID = 'example_redshift_cluster'
 REDSHIFT_CLUSTER_IDENTIFIER = getenv("REDSHIFT_CLUSTER_IDENTIFIER", "redshift-cluster-1")
 
 with DAG(
```

Comment:
Did you try running this test dag? At the moment tests with Sensors do not run successfully (I'm working on a fix) so I'd be surprised if this test passed?